LiskHQ/lisk-hub
# Refactor Price chart API call to redux action - Closes #293

### What was the problem?
- currencyGraph is not using store to handle data updates
### How did I fix it?
- add new reducer,actions liskService to handle request in redux state
- updated tests which were breaking after refactor
### How to test it?
 - go to `/dashboard` and check redux console
- `prices` should be saved on store, and new actions are triggered when opening the dashboard

### Review checklist
- The PR solves #293 
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
